Project/Assets/Prairie/Framework/Script/Interaction/Slideshow.cs

```diff
@@ -1,6 +1,7 @@
 using UnityEngine;
 using System.Collections;
 
+[AddComponentMenu("Prairie/Interactions/Slideshow")]

```

Comment:
I'd forgotten that it was triggered by the annotation button. That could be a little confusing, and we may want to discuss it again at a meeting, but for now I will move it over to Annotation for now.